cryptography/src/bls12381/dkg/arbiter.rs

```diff
@@ -102,9 +102,19 @@ impl<P: PublicKey, V: Variant> Arbiter<P, V> {
         }
     }
 
-    /// Disqualify a participant from the DKG for external reason (i.e. sending invalid messages).
-    pub fn disqualify(&mut self, participant: P) {
-        self.disqualified.insert(participant);
+    /// Disqualify a dealer from the DKG for external reason (i.e. sending invalid messages).
+    ///
+    /// If the participant is not a dealer, this is a no-op.
+    ///
+    /// # Warning
+    ///
+    /// If the [Arbiter] is being run by all participants, all participants must disqualify the
+    /// same public keys (or else will derive different group polynomials).
+    pub fn disqualify(&mut self, dealer: P) {
```

Comment:
We could return a result here (with InvalidDealer)?

Reply:
Would be nice to signal misuse, though I imagine users would also treat that error as a no-op. It would give them the chance to log, though.